Package_SON.pretty/VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm.kicad_mod

```diff
@@ -0,0 +1,44 @@
+(module VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm (layer F.Cu) (tedit 5B9F93A0)
```

Comment:
The name should be VSON-10-1EP_3x3mm_P0.5mm_EP**1x2.1mm**  - EP dimensions should be listed in `X`,`Y` order. Also, consider adding the component height of 1mm.